tests/test_10_use_cases.py

```diff
@@ -0,0 +1,342 @@
+        h1.cmd('ip addr add 10.1.1.1/24 dev vlan100')
+        h6.cmd('ip link add link %s name vlan100 type vlan id 100' % (h6.intfNames()[0]))
+        h6.cmd('ip link set up vlan100')
+        h6.cmd('ip addr add 10.1.1.6/24 dev vlan100')
+
+        # test connectivity
+        assert ', 0% packet loss,' in h1.cmd('ping -c4 10.1.1.6')
+
+        Tenet01 = self.net.net.get('Tenet01')
+        Tenet01.intf('Tenet01-eth50').ifconfig('down') 
+
+        time.sleep(15)
+
+        api_url = SDX_CONTROLLER + '/l2vpn/1.0'
+        data = requests.get(api_url).json()
+        assert data[key]["status"] == "down"  ### FAIL HERE
+
+        # test connectivity
+        assert ', 100% packet loss,' in h1.cmd('ping -c4 10.1.1.6')
+
+        ### Reset 
+        Tenet01.intf('Tenet01-eth50').ifconfig('up') 
+
+        time.sleep(15)
+
+        data = requests.get(api_url).json()
+        assert data[key]["status"] == "up"
+
+        # test connectivity
+        assert ', 0% packet loss,' in h1.cmd('ping -c4 10.1.1.6')
+
+    def test_025_update_with_node_down(self):
+        ''' Use case 4
+            OXPO sends a topology update with a Node down (switch down).
+        '''
+        def set_node(node, status, target):
+            if status == 'down':
+                node.cmd(f"ovs-vsctl set-controller {node.name} {target}")
+                node.cmd(f"ovs-vsctl get-controller {node.name}") 
+            else:
+                node.cmd(f"ovs-vsctl set-controller {node.name} {target}")
+                node.cmd(f"ovs-vsctl get-controller {node.name}") 
+
+        key = self._create_l2vpn("urn:sdx:port:ampath.net:Ampath1:50","urn:sdx:port:tenet.ac.za:Tenet01:50")
+
+        h1, h6 = self.net.net.get('h1', 'h6')
+        h1.cmd('ip link add link %s name vlan100 type vlan id 100' % (h1.intfNames()[0]))
+        h1.cmd('ip link set up vlan100')
+        h1.cmd('ip addr add 10.1.1.1/24 dev vlan100')
+        h6.cmd('ip link add link %s name vlan100 type vlan id 100' % (h6.intfNames()[0]))
+        h6.cmd('ip link set up vlan100')
+        h6.cmd('ip addr add 10.1.1.6/24 dev vlan100')
+
+        # test connectivity
+        assert ', 0% packet loss,' in h1.cmd('ping -c4 10.1.1.6')
+
+        api_url = SDX_CONTROLLER + '/l2vpn/1.0'
+        data = requests.get(api_url).json()
+
+        api_url_topology = SDX_CONTROLLER + '/topology'
+        response = requests.get(api_url_topology)
+        data = response.json()
+        status_nodes = {node['name']:node['status'] for node in data['nodes']}
+        assert status_nodes['Ampath1'] == 'up'
+
+        Ampath1 = self.net.net.get('Ampath1')
+        set_node(Ampath1, 'down', "tcp:127.0.0.1:6654")
+
+        time.sleep(15)
+
+        response = requests.get(api_url_topology)
+        data = response.json()
+        status_nodes = {node['name']:node['status'] for node in data['nodes']}
+        assert status_nodes['Ampath1'] == 'down'
+
+        ### Reset
+        set_node(Ampath1, 'up', "tcp:127.0.0.1:6653")
```

Comment:
this is not the correct IP address for the controller at Ampath OXP. I suggest that you get the current IP before setting to something else and then restore the previous config here.